patches/disable-downloads.diff

```diff
@@ -78,14 +78,14 @@ Index: code-server/lib/vscode/src/vs/server/node/webClientServer.ts
 ===================================================================
 --- code-server.orig/lib/vscode/src/vs/server/node/webClientServer.ts
 +++ code-server/lib/vscode/src/vs/server/node/webClientServer.ts
-@@ -290,6 +290,7 @@ export class WebClientServer {
+@@ -293,6 +293,7 @@ export class WebClientServer {
  					logLevel: this._logService.getLevel(),
  				},
  				userDataPath: this._environmentService.userDataPath,
 +				isEnabledFileDownloads: !this._environmentService.args['disable-file-downloads'],
  				settingsSyncOptions: !this._environmentService.isBuilt && this._environmentService.args['enable-sync'] ? { enabled: true } : undefined,
- 				productConfiguration: <Partial<IProductConfiguration>>{
- 					rootEndpoint: base,
+ 				enableWorkspaceTrust: !this._environmentService.args['disable-workspace-trust'],
+ 				folderUri: resolveWorkspaceURI(this._environmentService.args['default-folder']),
```

Comment:
I assume this just got moved around a bit

Reply:
Yeah looks like two new lines were added which pushed down the old stuff out of the patch context.